internal/rbd/rbd_util.go

```diff
@@ -1002,7 +1010,7 @@ func (rv *rbdVolume) cloneRbdImageFromSnapshot(ctx context.Context, pSnapOpts *r
 	}
 
 	util.DebugLog(ctx, logMsg,
-		pSnapOpts, image, rv.imageFeatureSet.Names(), rv.Monitors)
+		parentVol, pSnapOpts.String(), rv, rv.imageFeatureSet.Names(), rv.Monitors)
```

Comment:
logging `parentVol` should not be required, it is supposed to be part of `pSnapOpts.String()`